threeplay.go

```diff
@@ -36,6 +38,18 @@ type File struct {
 	ErrorDescription     string `json:"error_description"`
 }
 
+type FilesPage struct {
+	Files   []File `json:"files"`
+	Summary `json:"summary"`
+}
+
+type Summary struct {
+	CurrentPage  json.Number `json:"current_page"`
```

Comment:
Ugh. Im afraid not :(

You need to be able to represent two different json types in a single Go copy you need a custom type.